discussion.php

```diff
@@ -25,6 +25,7 @@
 // Include config and locallib.
 require_once('../../config.php');
 require_once($CFG->dirroot . '/mod/moodleoverflow/locallib.php');
+require_once($CFG->dirroot . '/course/modlib.php');
```

Comment:
Why did you insert that?

Reply:
that was a temporary import, i removed it